src/index.js

```diff
@@ -30,24 +30,10 @@ parseHydraDocumentation(program.args[0]).then(api => {
       if (null === resourceToGenerate || nameLc === resourceToGenerate || titleLc === resourceToGenerate) {
         generator.generate(api, resource, program.args[1]);
 
-        const titleUcFirst = resource.title.charAt(0).toUpperCase() + resource.title.slice(1);
-
-        console.log('Code for the "%s" resource type has been generated!', resource.title);
-        console.log('Paste the following routes definitions in your application configuration:');
```

Comment:
Can you update this message instead of deleting it totally? The routes still need to be imported.